sources/advanced/Cli.ts

```diff
@@ -276,7 +276,7 @@ export class Cli<Context extends BaseContext = BaseContext> implements MiniCli<C
                     for (let [description, example] of examples) {
                         result += `\n`;
                         result += formatMarkdownish(description, false);
-                        result += (<string[]>[]).concat(example.replace(/^/m, `  ${chalk.bold(prefix)}${this.binaryName} `)).join(`\n`) + `\n`;
+                        result += (<string[]>[]).concat(example.replace(/\$0/m, `  ${chalk.bold(prefix)}${this.binaryName}`)).join(`\n`) + `\n`;
```

Comment:
I think the `prefix` should still be only put at the front so that things like `$0 workspaces list | $0 run something` become possible 🤔

Looking at the existing code I'm also not sure why I put this weird `concat` since the parameter is obviously a string (since we call `replace`) and never contains more than one element (so the `join` is useless). We probably can remove it if TS is happy with it.

```suggestion
                        result += example
                            .replace(/^/m, `  ${chalk.bold(prefix)}`)
                            .replace(/\$0/m, this.binaryName)
                        ) + `\n`;
```